README.md

```diff
@@ -100,3 +100,38 @@ $cat shibboleth/overrides/etc/shibboleth/metadata-providers.xml
 </MetadataProviders>
 ```
 configures 2 metadata sources
+
+# Case study - local testing
+
+1. create .env
+   ```
+   SHIB_HOSTNAME=jm
+   NGINX_TOMCAT=dev-5.pc:85
+   NGINX_NODE=dev-5.pc:85
+   NGINX_SERVER_NAMES="jm localhost"
+   NGINX_RESOLVERS="1.1.1.1 8.8.8.8"
+   NGINX_MAX_BODY_SIZE=10G
+   ```
+2. in ./nginx/ssl
+   ```
+   openssl dhparam -out dhparam.pem 4096
+   openssl genrsa -out /out/serverkey.pem 2048
+   openssl req -new -x509 -key /out/serverkey.pem -out /out/servercert.pem -days 365
+   cp ./servercert.pem ./nginx_chain_cert.pem
+   ```
+   (on win, you can use `docker run -it --rm  -v %cd%:/out nginx /bin/bash -c "CMD"`)
+3. in ./
+   ```
+   mkdir -p ./shibboleth/overrides/etc/shibboleth/
+   curl https://www.eduid.cz/docs/metadata2021.eduid.cz.crt.pem -o ./shibboleth/overrides/etc/shibboleth/metadata2021.eduid.cz.crt.pem
+   ```
+3. in ./
+   ```
+   docker compose up
+   ```
+5. open https://jm/Shibboleth.sso/DiscoFeed
+6. optionally, you can inspect logs, unless you changed the default contents or created `proxied-idp.xml`, there will be errors/warnings in the log
```

Comment:
The `additional configs for shibboleth` and its overrides section tries to document how to configure your own MetadataProvider(s)...i.e. it expects you to modify the example. If that's not clear, I need to word it in a better way...

Reply:
This comment was meant for the situation where someone uses `metadata-providers.xml` from the README.md but not create proxied-idp.xml, you will get 
```
error building MetadataProvider: Unable to access local file (/opt/shibboleth-sp/var/cache/shibboleth/proxied-idp.xml)
```